Update removing-user-from-restricted-users-portal-after-spam.md (#714)
MicrosoftDocs/OfficeDocs-o365seccomp: SecurityCompliance/removing-user-from-restricted-users-portal-after-spam.md

## Making sure admins are alerted when this happens

A "User restricted from sending email" alert is available as a policy under the Office 365 Security & Compliance Alert policies page. This was formerly the outbound spam policy but is now native to the Office 365 alerting platform. Go to [Alert policies in the Security & Compliance Center](alert-policies.md) for more information on alerts.

> [!IMPORTANT]

> > In order for alerts to work, audit log search needs to be turned on. More information can be found in [Turn Office 365 audit log search on or off](turn-audit-log-search-on-or-off.md)

Reviewer: ```suggestion
> For alerts to work, audit log search must to be turned on. See how to [Turn Office 365 audit log search on or off](turn-audit-log-search-on-or-off.md) for more information.
```
